nvflare/private/fed/client/client_runner.py

```diff
@@ -213,7 +214,7 @@ def _do_process_task(self, task: TaskAssignment, fl_ctx: FLContext, abort_signal
             if scope_object.task_data_filters:
                 filter_list.extend(scope_object.task_data_filters)
 
-        task_filter_list = self.task_data_filters.get(task.name)
+        task_filter_list = self.task_data_filters.get(task.name + FilterChain.DELIMITER + FilterChain.IN)
         if task_filter_list:
             filter_list.extend(task_filter_list)
```

Comment:
can we make client runner also reuse `apply_data_filters` and `apply_result_filters` somehow?